docs/sources/administration/preferences/_index.md

```diff
@@ -0,0 +1,20 @@
++++
+title = "Preferences"
+aliases =["/docs/grafana/latest/administration/preferences.md"]
+weight = 50
++++
+
+# Grafana preferences
+
+Grafana preferences are basic settings. They control your Grafana theme, your home dashboard, your time zone, and so on.
+
+Preferences are sometimes confusing, because they can be set at four different levels. Listed in order from highest to lowest, they are:
```

Comment:
```suggestion
Preferences are sometimes confusing because they can be set at four different levels according to highest-to-lowest precedence:
```
Rephrased to use a complete sentence.